examples/stun/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	"github.com/pions/webrtc"
+)
+
+func main() {
+	peerConnection, err := webrtc.New(&webrtc.RTCConfiguration{
+		ICEServers: []webrtc.RTCICEServer{
+			{
+				URLs: []string{"stun:stun.l.google.com:19302"},
+			},
+		},
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	if err := peerConnection.CreateAnswer(); err != nil {
```

Comment:
Love this example! I would dump the session description maybe? The user can see their public IP might be helpful to see the difference.

Reply:
gotcha